sdk/cosmos/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -199,22 +199,19 @@ public void testUpsertNewDocument() {
     public void testUpdateWithReturnEntity() {
         final Person updated = new Person(TEST_PERSON.getId(), UPDATED_FIRST_NAME,
             TEST_PERSON.getLastName(), TEST_PERSON.getHobbies(), TEST_PERSON.getShippingAddresses());
-        updated.set_etag(insertedPerson.get_etag());
 
```

Comment:
oops this should not have been removed. Forgot i had renamed this field. Ill add this back

Reply:
Yes, please add these asserts back.

Reply:
fixed